activities/communication/blogging.md

```diff
@@ -91,7 +91,10 @@ Example:
 Commit your changes and create a pull request.
 Tag the communications coordinator plus any other colleagues whose review you'd like on the pull request for your blogpost for a final proofread.
 
-Once approved, your pull request will be merged and published.
+Once approved, your pull request will be merged.
+If the date in the frontmatter was the same (or earlier) than the day the pull request was merged it will appear as published in the blog immediately.
```

Comment:
```suggestion
If the date in the frontmatter is the same as (or earlier than) the day the pull request was merged it will appear as published in the blog immediately.
```